src/main/java/org/reaktivity/nukleus/socks/internal/stream/SocksServerFactory.java

```diff
@@ -0,0 +1,800 @@
+
+        private void onNetworkBegin(
+            BeginFW begin)
+        {
+            doNetworkBegin(supplyTraceId.getAsLong());
+        }
+
+        private void onNetworkEnd(
+            EndFW end)
+        {
+            final long traceId = end.trace();
+            doNetworkEnd(traceId);
+        }
+
+        private void onNetworkAbort(
+            AbortFW abort)
+        {
+            final long traceId = abort.trace();
+            doNetworkAbort(traceId);
+        }
+
+        private void onNetworkWindow(
+            WindowFW window)
+        {
+            final int replyCredit = window.credit();
+
+            replyBudget += replyCredit;
+            replyPadding += window.padding();
+
+            final int initialCredit = bufferPool.slotCapacity() - initialBudget;
+            doNetworkWindow(supplyTraceId.getAsLong(), initialCredit);
+        }
+
+        private void onSocksRequest(
+            SocksRequestFW socksRequestFW)
+        {
+            if (socksRequestFW.version() != 5 && socksRequestFW.reserved() != 0)
+            {
+                //TODO
+            }
+
+            SocksCommandType commandType = socksRequestFW.command().get();
+            decodeCommandType(commandType,
+                socksRequestFW.buffer(),
+                socksRequestFW.offset(),
+                socksRequestFW.limit());
+        }
+
+        private void onSocksConnect(
+            SocksRequestFW socksRequest)
+        {
+            if (socksRequest == null)
+            {
+                //TODO
+            }
+
+            final MessagePredicate filter = (t, b, o, l) ->
+            {
+                if (t > 0 && t <= SignalFW.TYPE_ID && o < l)
+                {
+                    return true;
+                }
+                return false;
+            };
+
+            final RouteFW route = router.resolve(routeId, this.authorization, filter, wrapRoute);
+            if(route != null)
+            {
+                final long newRouteId = route.correlationId();
+                final long newInitialId = supplyInitialId.applyAsLong(newRouteId);
+                final long newReplyId = supplyReplyId.applyAsLong(newInitialId);
+
+                final MessageConsumer newTarget = router.supplyReceiver(newInitialId);
+                final SocksServerStream socksServerStream = new SocksServerStream(this, newTarget,
+                    newRouteId, newInitialId, newReplyId);
+
+                SocksBeginExFW socksBeginEx = socksBeginExRW.wrap(extBuffer,0, extBuffer.capacity())
+                                                            .typeId(socksTypeId)
+                                                            .address(socksRequest.address().domainName())
+                                                            .port(socksRequest.port())
+                                                            .build();
+
+                socksServerStream.doApplicationBeginEx(newTarget, newInitialId, decodeTraceId,
```

Comment:
Rename `doApplicationBeginEx` to `doApplicationBegin` and move `socksBeginEx` lines above inside `doApplicationBegin` instead.

Reply:
If I move everything in one line, it is going to be very long, here is what I did: [https://github.com/JinL0/nukleus-socks.java/blob/81cc6deb7f278c4f279767070384c76e3556de2f/src/main/java/org/reaktivity/nukleus/socks/internal/stream/SocksServerFactory.java#L417](url)